h/indexer/__init__.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
```

Comment:
Probably worth correcting the commit message to say "package" rather than "module".